x/lightclient/types/message_disable_verification_flags.go

```diff
@@ -46,21 +44,12 @@ func (msg *MsgDisableHeaderVerification) ValidateBasic() error {
 	if _, err := sdk.AccAddressFromBech32(msg.Creator); err != nil {
 		return cosmoserrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
 	}
-	chainListForHeaderSupport := chains.ChainListForHeaderSupport()
 	if len(msg.ChainIdList) == 0 {
 		return cosmoserrors.Wrapf(sdkerrors.ErrInvalidRequest, "chain id list cannot be empty")
 	}
-	if len(msg.ChainIdList) > len(chainListForHeaderSupport) {
-		return cosmoserrors.Wrapf(sdkerrors.ErrInvalidRequest, "chain id list cannot be greater than supported chains")
-	}
-	for _, chainID := range msg.ChainIdList {
-		if !chains.ChainIDInChainList(chainID, chainListForHeaderSupport) {
-			return cosmoserrors.Wrapf(
-				sdkerrors.ErrInvalidRequest,
-				"invalid chain id header not supported (%d)",
-				chainID,
-			)
-		}
+
+	if len(msg.ChainIdList) > 200 {
```

Comment:
What does that mean?

Reply:
This is an arbitrary value as a static check, as the value is stored as a singleton state. Can add comment for this one

Reply:
a7af5d8c6